apps/vmq_server/src/vmq_metrics.erl

```diff
@@ -645,6 +648,24 @@ incr_topic_counter(Metric) ->
             end
     end.
 
+-spec incr_matched_topic(binary() | undefined, atom(), integer()) -> ok.
+incr_matched_topic(<<>>, _Type, _Qos) ->
+    ok;
+incr_matched_topic(undefined, _Type, _Qos) ->
+    ok;
+incr_matched_topic(Name, Type, Qos) ->
+    OperationName =
+        case Type of
+            read -> subscribe;
+            write -> publish;
+            _ -> Type
+        end,
+    _ = incr_topic_counter(
```

Comment:
What is the use of `_ =`?

Reply:
Removed in https://github.com/gojek/vernemq/pull/42/commits/cb7bf14297faa5490b6fa7f62c136725425e5991